modules/data.atmosphere/man/download.FluxnetLaThuile.Rd

```diff
@@ -17,7 +17,9 @@ download.FluxnetLaThuile(
 }
 \arguments{
 \item{sitename}{the FLUXNET ID of the site to be downloaded, used as file name prefix.
-The 'SITE_ID' field in \href{http://www.fluxdata.org/DataInfo/Dataset\%20Doc\%20Lib/SynthDataSummary.aspx}{list of Fluxnet LaThuile sites}}
+% The 'SITE_ID' field in \href{http://www.fluxdata.org/DataInfo/Dataset\%20Doc\%20Lib/SynthDataSummary.aspx}{list of Fluxnet LaThuile sites}}
```

Comment:
All our Rd files are generated by Roxygen, so please don't edit them by hand. Instead edit the Roxygen comments in the appropriate section of `modules/data.atmosphere/R/download.FluxnetLaThuile.R` and the changes will propagate here when you urn Roxygen.

Reply:
https://fluxnet.org/data/la-thuile-dataset/ is probably the URL to use there